### PR TITLE
global: invenio-collections initial usage

### PR DESCRIPTION
**Pull request by jmartinm**
Signed-off-by: Javier Martin Montull <javier.martin.montull@cern.ch>

### CODE REVIEW

**Comment by jmartinm** [comment]
Had to remove [this query](https://github.com/inspirehep/inspire-next/blob/production/inspirehep/base/mixer/sources/collection.json#L78) as the parser was giving an error on the wildcard query.

**Comment by jmartinm** [comment]
The errors on Travis are happening because when enabling the search enhancer of `invenio-collections` the search query gets enriched with the `_collections` field. I guess we will just need to update the tests.

File: inspirehep/modules/search/query.py [hunk 1]
```diff
@@ -40,15 +40,22 @@
...
 
 class InspireQuery(Query):
+
     """Extension of invenio_search.api.Query."""
 
     def __init__(self, *args, **kwargs):
         """Provide Query parameters."""
+        self.collection = kwargs.pop(
+            "collection",
+            current_app.config['THEME_SITENAME']
```
Comment (kaplun) [inline inspirehep/modules/search/query.py]: Uh? Can you document what is this magic? :)

**Comment by kaplun** [comment]
We need to reopen this PR against `master`

**Comment by jmartinm** [comment]
superseded by https://github.com/inspirehep/inspire-next/pull/1116

**Pull request closed without merging**